subsys/net/openthread/rpc/server/ot_rpc_message_server.h

```diff
@@ -14,5 +14,6 @@
 ot_msg_key ot_reg_msg_alloc(void);
 void ot_msg_free(ot_msg_key key);
 otMessage *ot_msg_get(ot_msg_key key);
+bool ot_msg_set(ot_msg_key key, otMessage *msg);
```

Comment:
`ot_reg_msg_alloc` could take `otMessage*` as a parameter and then this wouldn't be needed.

Reply:
fixed.